### DISCUSSION TITLE
Swiss WRF Model

### DISCUSSION THREAD

**Discussion by jortaca** [open-meteo/open-meteo#1108] [General]
The Swiss WRF model was the most accurate in predicting the recent “DANA” (Isolated Depression at High Levels) in Valencia. What insights do you have about this model? Is it freely accessible, and could it potentially be integrated into Open-meteo?

I’ve found very little information about this model online, and I’ve heard it’s considered experimental, meaning its data should be treated as such. But then again, aren’t all models experimental to some extent? Additionally, I read it has a 1 km resolution, which makes it quite intriguing.

What are your thoughts? What pros and cons do you see in using it?

**Comment by patrick-zippenfenig** [comment]
I am only aware of the Swiss COSMO model that is soon being replaced with an ICON model with an extension to run on a GPU cluster. In one configuration it runs at 1 km, but as far as I know, the results are not better than courser 2-3 km resolution models.

Although Switzerland is moving towards open-data, this transition still takes time and it is unknown when numerical forecasts will be available as open-data. I am in contact with the open-data team as MeteoSwiss, but there is nothing I can share.
